eth/executionclient/execution_client.go

```diff
@@ -294,22 +287,7 @@ func (ec *ExecutionClient) Healthy(ctx context.Context) error {
 		return nil
 	}
 
-	ec.healthyChMu.Lock()
-	defer ec.healthyChMu.Unlock()
-
-	if err := ec.healthy(ctx); err != nil {
-		close(ec.healthyCh)
```

Comment:
This triggers closing of closed channel which panics, I'm removing it as it's dangerous to use
FYI @moshe-blox